packages/@okta/vuepress-site/docs/reference/api/threat-insight/index.md

```diff
@@ -5,17 +5,18 @@ category: management
 
 # ThreatInsight configuration API
 
-[Okta ThreatInsight](https://help.okta.com/okta_help.htm?id=ext_threatinsight) maintains a constantly evolving list of IPs that exhibit suspicious behaviors suggestive of malicious activity. Authentication requests associated with an IP in this list can be logged in [System Log](https://help.okta.com/okta_help.htm?id=ext_Reports_SysLog) and blocked. The Okta ThreatInsight Configuration API provides operations to manage your ThreatInsight configuration.
+[Okta ThreatInsight](https://help.okta.com/okta_help.htm?id=ext_threatinsight) maintains a constantly evolving list of IP addresses that consistently exhibit malicious activity. Authentication requests associated with an IP in this list can be logged in [System Log](https://help.okta.com/okta_help.htm?id=ext_Reports_SysLog) and blocked. ThreatInsight also covers non-authentication requests depending on the attack patterns of these malicious IPs.
```

Comment:
should we specify that we cover non-authentication requests `in limited capacity`? I think thats what you are trying to convey by adding `depending on the attack patterns of these malicious IPs`.  

So, maybe --
`ThreatInsight also covers non-authentication requests in limited capacity depending on the attack patterns of these malicious IPs`

Even the current version looks good as well. Thanks!

Reply:
Thanks @santoshmale-okta , I'll update the docs to your version. And updated the oas3 version as well. Thanks